wfl/Controllers/RestaurantsController.cs

```diff
@@ -42,7 +42,11 @@ public RestaurantsController(IDBManager dbManager)
         [HttpGet]
         public async Task<IHttpActionResult> GetAsync()
         {
-            return Ok(await DBManager.GetAllRestaurantsAsync());
+            RestaurantCollection restaurantCollection = new RestaurantCollection()
```

Comment:
Why didn't we simply return the enumerable restaurants item?  Curious why we created the wrapping RestaurantCollection object?